bin/mount

```diff
@@ -20,11 +20,14 @@ case FreeBSD
 	echo 'don''t know how to mount (no fuse)' >[1=2]
 case Darwin
 	version=`{sw_vers -productVersion|cut -d. -f1,2}
+	major_version = `{echo $version|cut -d. -f1}
 	if(sysctl fuse.version >[2]/dev/null |9 grep -si 'fuse.version' ||
 	   sysctl macfuse.version.number >[2]/dev/null |9 grep -si 'fuse.version' ||
 	   sysctl osxfuse.version.number >[2]/dev/null |9 grep -si 'fuse.version' ||
 	   test -d /System/Library/Extensions/fusefs.kext ||
 	   test -d /Library/Filesystems/osxfuse.fs/Contents/Extensions/$version/osxfuse.kext ||
+	   test -d /Library/Filesystems/macfuse.fs/Contents/Extensions/$version/macfuse.kext ||
+	   test -d /Library/Filesystems/macfuse.fs/Contents/Extensions/$major_version/macfuse.kext ||
```

Comment:
MacFUSE has a folder `11` but not `11.1` like it does for 10.x, using the major version enables macOS >=11 to work.